keylime-ansible-gcp/roles/keylime-gcp/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
The `keylime-gcp` role doesn't look very GCP specific to me. How different is it from the stuff already in https://github.com/keylime/ansible-keylime ? Can we re-use that stuff?

Reply:
The role isn't GCP specific, but I believe it differs from the ansible-keylime role due to the inclusion of the rust agent + its dependencies.

Reply:
I would suggest updating that other role instead then for a couple of reasons. First, the rust agent will soon be the official agent, so developers should be using/building it. And second, it would be a real pain to try and maintain 2 different but very similar ansible roles.

Reply:
Sounds good, I will remove the keylime-gcp role and have this repository point to ansible-keylime for install. I can adjust ansible-keylime to support the rust agent and submit a PR there.

Reply:
Hooray it's done https://github.com/keylime/ansible-keylime/pull/25